samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -197,22 +215,116 @@ int main(void)
 			net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
 			sys_put_le32(iso_send_count, iso_data);
 			net_buf_add_mem(buf, iso_data, sizeof(iso_data));
-			ret = bt_iso_chan_send(&bis_iso_chan[chan], buf, seq_num);
+
+			/* Read Tx Sync interface returns error if no ISO data was given previously.
+			 */
+			ret = bt_iso_chan_get_tx_sync(&bis_iso_chan[chan], &tx_info);
+			if (ret) {
+				printk("Failed to get tx sync before send (%d)\n", ret);
+
+				/* Use sequence number 0, to synchronize current stream with the
+				 * current payload counter in the ISO group of streams.
+				 */
```

Comment:
The spec doesn't state that the sequence number shall be used to find the corresponding payload counter. The controller is free to ignore it completely or only ignore it when it seems wrong.

I would therefore consider updating the comment:
```suggestion
				/* Use sequence number 0 for the first SDU sent on a given stream.
				 */
```
More information could be put in the readme of the sample.